apps/api/src/server/repositories/folder.repository.js

```diff
@@ -1,12 +1,72 @@
 import { databaseConnector } from '../utils/database-connector.js';
 
+/** @typedef {import('@pins/api').Schema.Folder} Folder */
+/** @typedef {import('@pins/applications').FolderDetails} FolderDetails */
+
+/**
+ * Returns array of folders in a folder or case (if parentFolderId is null)
+ *
+ * @param {number} caseId
+ * @param {number |null} parentFolderId
+ * @returns {Promise<Folder[]>}
+ */
+export const getByCaseId = (caseId, parentFolderId) => {
+	// if no parentFolderId, explicitly set null value im the call, to get the top level folders on the case
+	return parentFolderId
```

Comment:
it would read better if you set a `const parentId = parentFolderId || null;`  and then `return databaseConnector.folder.findMany({ where: { caseId, parentId } });`

Reply:
agreed - I didn;t really like the code I had written, will do.